prov/shm/src/smr_ep.c

```diff
@@ -949,8 +920,11 @@ static int smr_ep_close(struct fid *fid)
 
 	ofi_endpoint_close(&ep->util_ep);
 
-	if (ep->region)
+	if (ep->region) {
+		assert(ofi_atomic_read64(&ep->region->smr_cnt) ==
```

Comment:
prov/shm/src/smr_ep.c:924:41: error: 'struct smr_region' has no member named 'smr_cnt'; did you mean 'cmd_cnt'?
   assert(ofi_atomic_read64(&ep->region->smr_cnt) ==

Reply:
> prov/shm/src/smr_ep.c:924:41: error: 'struct smr_region' has no member named 'smr_cnt'; did you mean 'cmd_cnt'? assert(ofi_atomic_read64(&ep->region->smr_cnt) ==

I think that's due to the patch not being rebased to the latest code. I'll update